.github/workflows/openms-ci.yml

```diff
@@ -120,7 +121,8 @@ jobs:
           BUILD_TYPE: "Release"
           OPENMP: "Off"
           USE_STATIC_BOOST: "On"
-          BUILD_FLAGS: "-p:CL_MPCount=2"
+          #  BUILD_FLAGS: "-p:CL_MPCount=2" # For VS Generator and MSBuild
+          BUILD_FLAGS: "-j2" # For Ninja
```

Comment:
```suggestion
          BUILD_FLAGS: "-j2" # Ninja will otherwise use all cores (doesn't go well in GHA) 
```

Reply:
how about the other occurrences that are there for years?